tests/47.t

```diff
@@ -0,0 +1,9 @@
+description: select the first choice
+keys: \033OH \\n # HOME ENTER
```

Comment:
Could we move to the second choice prior pressing `HOME`, just to make
sure it actually works:

```
keys: \016 \033OH \\n # DOWN HOME ENTER
```